powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/InitActivationRequest.java

```diff
@@ -40,7 +41,9 @@ public class InitActivationRequest {
     private String applicationId;
     private Date timestampActivationExpire;
     private Long maxFailureCount;
+    @Deprecated
```

Comment:
Maybe we may add Javadoc, something like
```suggestion
    /**
    * @deprecated use {@link #commitPhase} instead
    **/
    @Deprecated
```